### ISSUE TITLE
npm install fails

### ISSUE THREAD

**Issue by invegat** [facebook/draft-js#1137]
[20:44:08] 'check-dependencies' errored after 5.32 s
[20:44:08] Error in plugin 'check-dependencies'
Message:
    npm broke
[20:44:08] 'default' errored after 5.33 s
[20:44:08] Error in plugin 'run-sequence(check-dependencies)'
[npm-debug.log.txt](https://github.com/facebook/draft-js/files/921186/npm-debug.log.txt)

**Comment by hanyulo** [comment]
Ran into same problem here.

**Comment by hanyulo** [comment]
@invegat  You can check this [#914] (https://github.com/facebook/draft-js/issues/914). Basically, it is node/npm version problem.

**Comment by invegat** [comment]
Edited my local /fbjs-scripts/gulp/check-dependencies.js L38 to 
- if (code !== 0 && code !== 1) {

and changed package.json 
-   "devEngines": {
    "node": "4.x || 6.x || 7.x",
    "npm": "2.x || 3.x || 4.x"
  },

was able to run the package.json scripts

Was unable to run the webside folder project in Windows 10, "npm start" errored out.  Got it working on 
the Windows 10 Bash Shell Linex with
-   [Xming](https://www.howtogeek.com/261575/how-to-run-graphical-linux-desktop-applications-from-windows-10s-bash-shell/)
-   [FireFox for Linux](https://support.mozilla.org/en-US/kb/install-firefox-linux?cache=no)

**Comment by mitermayer** [comment]
cc @zpao can we increase node engine to 7.x ?

**Comment by zpao** [comment]
My general feeling is that we should ensure we support building with LTS supported versions. If it doesn't work with LTS versions (eg 7.x, 8.x prior to last week), then we shouldn't go out of our way to support it.

The other bit about code checking - that was fixed in a newer version of fbjs-scripts, along with some other things. So I recommend updating that as well. There's an immutable-related change. Not sure if it affects Draft, but should make sure everything is working post-update.

tldr; sure, and add 8 as well. Also, update fbjs-scripts (read that changelog)

**Comment by juanlanus** [comment]
I had the issue.  
According to  https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5  this issue is triggered by the deprecation of a module at Medium.

**Comment by davoscript** [comment]
Thanks @mrkev that did the trick. I was able to build and submit a pull request to fix another issue :)

**Comment by raelmiu** [comment]
Same issue. Used Yarn and that seemed to work.

**Comment by Marckon** [comment]
> I can't repro on `node 10.16.3` and `yarn 1.12.1`. Try using yarn. The repo doesn't even have a `package-lock.json` (I personally only use yarn for draft-js).

Thanks bro!
